src/kv/kv.h

```diff
@@ -3,6 +3,7 @@
 #pragma once
```

Comment:
What's the performance gain of using `DLList` over `std::list` in our kv benchmarks?

Reply:
I would also like to have a picobench for this, this is a non-trivial amount of code that replaces equivalent STL functionality, we ought to have some data to back up the change.

Reply:
At current master:

commit_latency:
===============================================================================
   Name (baseline is *)   |   Dim   |  Total ms |  ns/op  |Baseline| Ops/second
===============================================================================
         commit_latency * |      10 |     0.000 |       2 |      - |500000000.0
         commit_latency * |     100 |     0.000 |       0 |      - |5263157894.7
         commit_latency * |    1000 |     0.000 |       0 |      - |31250000000.0
===============================================================================


PR:

commit_latency:
===============================================================================
   Name (baseline is *)   |   Dim   |  Total ms |  ns/op  |Baseline| Ops/second
===============================================================================
         commit_latency * |      10 |     0.000 |       1 |      - |666666666.7
         commit_latency * |     100 |     0.000 |       0 |      - |5263157894.7
         commit_latency * |    1000 |     0.000 |       0 |      - |55555555555.6
===============================================================================